### DISCUSSION TITLE
whats up with the duplicate packages ?

### DISCUSSION THREAD

**Discussion by revelator** [msys2/MINGW-packages#10386] [Q&A] [answered]
f2c scotch etc. are these for clang ?

**Comment by MehdiChinoune** [comment]


**Reply by revelator** [reply]
i mean f2c has an offshot linking to blas lapack and scotch has a duplicate with 64 bit integers.
mingw-w64-f2c and mingw-w64-f2cblaslapack
mingw-w64-scotch and mingw-w64-scotch-int64

**Reply by revelator** [reply]
also noticed that openblas now builds 2 packages for 64 bit one with 32 bit integers and on using 64 bit ones.

**Reply by revelator** [reply]
sure though it does not tell me much :) only thing i was interrested in knowing was wether these are intended for clang or if there is another purpose.

**Comment by LABPBS** [comment, marked as answer]
f2cblaslapack seems to be an alternative implementation that doesnt require a fortran compiler (specifically for clang) added in #10199
the 64 bit stuff was requested in #9757 and added in #10094

**Reply by revelator** [reply]
ah cool :) thx for explaining.
